### ISSUE TITLE
WS2812 Add option to color a range of pixels

### ISSUE THREAD

**Issue by martin072** [arendst/Sonoff-Tasmota#1790]
Would it be possible to add an option to set a color to a range of pixels on a strip?
The command  'LEDx #xxxxxx ' will do it for only one pixel, but I would like to control a range of pixels on the strip like this 
(preferably with the nice color fading option as well, look really nice on the whole strip when changing colours). 

Thanks!
Martin

**Comment by arendst** [comment]
Command pixels lowers the amount of leds under control.

**Comment by martin072** [comment]
yes, but that will set 0..<num pixels> I would be looking for an option to set pixels 0-30 in one color, 31-60 in another and 61-90 off for example.

**Comment by arendst** [comment]
Too much hassle.

**Comment by arendst** [comment]
And then it is implemented in the latest development version. See also #2303
